mmdet3d/core/bbox/structures/coord_3d_mode.py

```diff
@@ -245,7 +253,7 @@ def convert_point(point, src, dst, rt_mat=None):
         else:
             xyz = arr[:, :3] @ rt_mat.t()
 
-        remains = arr[..., 3:]
```

Comment:
Is this change necessary?

Reply:
This is not in accordance with the others, so I unified it.

Reply:
if we want to support any shape of input pc, this function needs to be refactored